packages/dds/sequence-sequential-id/src/generateSequentialId.ts

```diff
@@ -0,0 +1,80 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { assert } from "@fluidframework/common-utils";
+
+export const digitZero = "A";
+export const digitLast = "Z";
+const idBase = 26;
+
+export function isValidSeqId(seqId: string): boolean {
+  for (let i = 1; i < seqId.length; i++) {
+    const letterCode = getLetterCode(seqId[i]);
```

Comment:
I think you can optimize it a bit. Given that you know there will be no complex characters (requiring multiple code points), you can just do

const base = digitZero.charCodeAt(0); // in global scope

const letterCode = char.charCodeAt(i) - base;

charCodeAt(index) -

Reply:
Refactored a bit to use base from global scope